mitmproxy/tools/console/statusbar.py

```diff
@@ -281,7 +283,8 @@ def get_status(self):
         if self.master.options.mode != "regular":
             r.append("[%s]" % self.master.options.mode)
         if self.master.options.scripts:
-            r.append("[scripts:%s]" % len(self.master.options.scripts))
+            scripts = list(chain.from_iterable([glob(re) for re in self.master.options.scripts]))
```

Comment:
This snippet seems to appear the second time now - maybe split it off into a function and reuse it?

Reply:
Is the diff wrong - or did you want to you the new function?

Reply:
Sorry, I forgot to fix this part. 
Now, it's fixed.